app/components/FoodDeck.jsx

```diff
@@ -38,32 +38,33 @@ class FoodDeck extends Component {
   }
 
   render() {
-
-    const {foodDeck} = this.props.gameState;
+    const { foodDeck } = this.props.gameState;
     const FACEUP_COUNT = 4;
     const faceup = foodDeck.slice(-FACEUP_COUNT);
     let remaining = foodDeck.length - FACEUP_COUNT;
     if (remaining < 0) remaining = 0;
 
-    const faceupList = faceup.map(c => <li key={c.id} className="faceup-item"><FoodCard dragType="buyCard" {...c} /></li>);
+    const faceupList = faceup.map(c => (
+      <li key={c.id} className="faceup-item">
+        <FoodCard dragType="buyCard" mana={this.props.gameState.mana} {...c} />
```

Comment:
Passing state mana to cards so they know to be buyable or not.

Reply:
A quick question on this, more for my own edification.  Is there any reason not to reach directly into gameState.mana from FoodCard.jsx?  This prevents us from having to "prime" all FoodCard instances with what is already a globally known value.

Reply:
I think this is an evolving idiom in the Redux community: how many components are "aware" and how many are not? Do we connect to the store everywhere such that Redux is now inextricable from the app or do we pass a lot of props around? etc etc. Basically, judgment call 😄 .